components/builder-api/src/server/services/memcache.rs

```diff
@@ -112,10 +112,10 @@ impl MemcacheClient {
         self.reset_namespace(&channel_ns_key(origin, channel));
     }
 
-    pub fn get_session(&mut self, user: &str) -> Option<Session> {
-        trace!("Getting session for user {} from memcached", user);
+    pub fn get_token(&mut self, token: &str) -> Option<Session> {
```

Comment:
Nitpicking a bit, but the "get_session" naming seems more accurate, and more consistent with the other function naming here (eg, get/set package)..